ios/gutenberg/GutenbergViewController.swift

```diff
@@ -24,6 +24,8 @@ class GutenbergViewController: UIViewController {
         gutenberg.delegate = self
         navigationController?.navigationBar.isTranslucent = false
         registerLongPressGestureRecognizer()
+
+        _ = try! FallbackJavascriptInjection(blockHTML: "Hello", userId: "1")
```

Comment:
This looks like it will need to be removed cc @etoledom

Reply:
`try!` will crash if there's any problem loading the required files. This is here as an "early alarm system" so we don't need to trigger the feature to catch it.

What do you think?

We can remove it or have a small comment on it.

Reply:
Sorry for the late reply, gotcha, sounds good to me!